src/bi-balance-rules.yaml

```diff
@@ -76,17 +63,6 @@ AFLD:
 	Targetable:
 		TargetTypes: GroundActor, C4, DetonateAttack, Structure, SPY.England.Infiltrate
 
-IRON:
-	Inherits@shape: ^2x2Shape
-	Valued:
-		Cost: 2000
-	Building:
-		Footprint: xx xx
```

Comment:
This looks like it's still needed compared with `bleed`:
```Footprint: ++ xx```

I guess the bounds below are still needed too, since I think the structure was shifted slightly too.

Reply:
it's implemented a bit differently https://github.com/OpenRA/OpenRA/pull/20832

Reply:
Ah - brilliant! Thanks.